README.md

```diff
@@ -86,15 +86,15 @@ and the data files in `/tmp/share`.
 
  ## Playing the game
  
- The game takes place on a rectangular areana where a snake continuously
+ The game takes place on a rectangular arena where a snake continuously
  move in one of the four directions: left, right, up and down --- it never 
- stops. As the snake moves it looses energy and if all of it is exausted, the
+ stops. As the snake moves, it looses energy, and if all this energy is exausted, the
  snake dies. To recover energy, the snake needs to eat pieces of food which
  are constantly replaced at random positions. 
```

Comment:
```suggestion
 are constantly placed at random positions. 
```

Reply:
Yeah makes sense. I don't think the food is replaced since a new one will be **placed** after the current is eaten (it's not implemented yet but I think it's how it'll work)